midAssessment/largest_number_array.rb

```diff
@@ -0,0 +1,22 @@
+# Develop a program to find the largest element in an array.
+def find_largest_element(arr)
+  if arr.empty?
+    return nil
+  end
+
+  largest_element = arr[0]
+
+  arr.each do |element|
+    if element > largest_element
+      largest_element = element
+    end
+  end
+
+  largest_element
+end
+
+# Example:
+array = [3, 7, 1, 9, 4, 6, 2, 8]
+result = find_largest_element(array)
```

Comment:
Can we do it without using loop?
Check reduce/inject method of array in Ruby.

Reply:
Added alternative solution using reduce method